code/Collapse.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from "react"
```

Comment:
This component still needs a lot of work to make it properly dynamic. There is a great example in the Framer Base Kit (list component) that uses ComponentArray and Stacks with dynamic height. Would be great to try and adapt it somehow to work with this and the Tab component.